<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrfJobsDialog</class>
 <widget class="QDialog" name="PrfJobsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPrf - Jobs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The following jobs are in the queue:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="stopSelected_pushButton">
       <property name="text">
        <string>&amp;Stop selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeStopped_pushButton">
       <property name="text">
        <string>&amp;Remove stopped</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
